### ISSUE TITLE
ASpace rejects Agent updates via API for records with identical titles but the same doesn't happen via UI

### ISSUE THREAD

**Issue by regineheberlein** [pulibrary/aspace_helpers#568]
When running Agents updates via API, some updates get rejected because the record's title string is identical to that for another record, even though both records are for distinct entities and other fields differ (e.g. dates).

The same does not happen when manually updating the records in the UI.

It seems that the API identifies duplicates on insufficient data points and in a different way from the UI? Investigate this behavior a bit further and, if applicable, submit as a bug report to ArchivesSpace.

**Comment by regineheberlein** [comment]
An example on staging:
`Bayard, James A. (James Asheton) (1767-1815)` (/agents/people/13167)
`Bayard, James A. (James Asheton), 1799-1880` (/agents/people/13168)

<img width="1430" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0RlQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f100801232070e8219bc28996963874ff18e5888/Screenshot%202024-09-17%20at%2010.58.54%E2%80%AFAM.png" alt="Screenshot 2024-09-17 at 10.58.54 AM.png" />

When I update the records via API in numerical sequence, the first record updates, but the second record throws a duplicate error:

```
{"status":"Updated","id":13167,"lock_version":12,"stale":true,"uri":"/agents/people/13167","warnings":[]}
{"error":{"names":["Agent must be unique"],"conflicting_record":["/agents/people/13167"]}}
```

**Comment by regineheberlein** [comment]
However, if the update of 13168 _did_ run, then the two records would be identical on:
- `primary_name`
- `rest of name`
- `fuller form`

(NB they would still differ on dates and `title`.)

<img width="1634" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzdmQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e382edab66ec905885c14fd2fefe8b3706c4df08/Screenshot%202024-09-17%20at%2011.32.50%E2%80%AFAM.png" alt="Screenshot 2024-09-17 at 11.32.50 AM.png" />

**Comment by regineheberlein** [comment]
Confirmed with Max that the duplicate validation relies on a hash consisting of the name field strings, three fields that are almost never populated (contact, external documents, and note), one constant (the date type "existence"), and one near constant (two possible values, "single" and "range"). All it take is for a record with no notes, contact information, or external documents to have the same name and the same date type (even if the date is different).

I've reported this to ArchivesSpace.
